File: DocumentGallery/DocumentGallery/app/entity/documentFolderEntity.ts
module DocumentGallery {
     interface IDcoumentGalleryEntity {
         Id: number;
         Name: string;
         Status_Id: number;
         Status_Name: string;
         Type: string;
         Parent_Id: number;
         UpdatedOn: Date;
    }

    export class DocumentFolderEntity {
        public Id: number;
        public Name: string;
        public Status_Id: number;
        public Status_Name: string;
        public Type: string;
        public Parent_Id: number;
        public UpdatedOn: Date;

        constructor() { }
    }
}

File: DocumentGallery/DocumentGalleryEntity/DocumentFolderEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DocumentGalleryEntity
{
    public class DocumentFolderEntity
    {
        public long? Id { get; set; }
        public long? Parent_Id { get; set; }
        public string Name { get; set; }
        public long? Size { get; set; }
        public DateTime UpdatedOn { get; set; }
        public string Type { get; set; }
        public long? Status_Id { get; set; }
        public string Status_Name { get; set; }

    }
}
